x-pack/plugins/observability/server/lib/rules/threshold/lib/metric_query.ts

```diff
@@ -156,7 +161,7 @@ export const getElasticsearchMetricQuery = (
       : void 0;
 
   const includesList = ['host.*', 'labels.*', 'tags', 'cloud.*', 'orchestrator.*'];
-  const excludesList = ['host.cpu.*', 'host.disk.*', 'host.network.*'];
+  const excludesList = ['host.cpu', 'host.disk', 'host.network'];
   if (!containerContextAgg) includesList.push('container.*');
```

Comment:
We should also exclude the metrics when `containerContextAgg` is not added.

<img width="602" alt="Screenshot 2023-07-20 at 13 00 20" src="https://github.com/elastic/kibana/assets/69037875/e3b06e75-dbca-4f9c-a258-1c4d50adeb93">

Reply:
Good catch, thanks!

Reply:
Fixed!